gobblin-modules/gobblin-kafka-common/src/main/java/org/apache/gobblin/source/extractor/extract/kafka/KafkaExtractorStatsTracker.java

```diff
@@ -107,6 +109,8 @@ public int getErrorPartitionCount() {
     private long startFetchEpochTime;
     private long stopFetchEpochTime;
     private long lastSuccessfulRecordHeaderTimestamp;
+    private long minLogAppendTime = -1L;
```

Comment:
Should be min initialized as LONG_MAX ?

Reply:
Want to leave it as -1 for the case where we are not tracking SLAs.